main.go

```diff
@@ -567,18 +726,37 @@ func main() {
 			Name:   "generate-markdown",
 			Hidden: true,
 		},
-		&cli.StringFlag{
-			Name:   "log-level",
-			Hidden: true,
-			Value:  "error",
-		},
 		&cli.BoolFlag{
 			Name:    "no-color",
 			Hidden:  false,
 			Value:   false,
 			EnvVars: []string{"NO_COLOR"},
 		},
+		&cli.StringFlag{
+			Name:      "config",
+			Hidden:    true,
+			Value:     defaultConfigFilePath,
+			TakesFile: true,
+			Usage:     "Read config values from `FILE`",
```

Comment:
What configuration file could be specified here? Is it rhsm.conf? Is it configuration for `rhc` or `rhcd` or `insights-client`?

Reply:
This is meant for rhc configuration file